docs/docs/getting-started/installation/configuration.md

```diff
@@ -127,6 +127,8 @@ components:
   integration:
     webhook:
       name: webhook
+    source-api:
+      enabled: true
```

Comment:
This should be under `components.api`, but we don't have such a section yet, correct?